hphp/test/slow/ext_string/setlocale-multiple-calls.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+test(LC_TIME, 'fr_FR');
```

Comment:
maybe use utf-8 locales. looks better in github and in terminal :)

`fr_FR.UTF-8`